Lib/uuid.py

```diff
@@ -66,6 +66,8 @@
 if _AIX:
     _MAC_DELIM = b'.'
     _MAC_OMITS_LEADING_ZEROES = True
+if sys.platform in ('darwin', ):
```

Comment:
```suggestion
if sys.platform == 'darwin':
```

Using `in` to test the value is not necessary at this moment, can be changed when there is another platform that needs this.

Reply:
Updated.